pyensign/events.py

```diff
@@ -0,0 +1,65 @@
+from pyensign.api.v1beta1 import event_pb2
+from pyensign.mimetype.v1beta1 import mimetype_pb2
+
+mimetypes = mimetype_pb2.DESCRIPTOR.enum_types_by_name["MIME"].values_by_name
+mimetype_names = mimetypes.keys()
+
+
+class Event:
+    """
+    Event is an abstraction of an Ensign protocol buffer Event to make it easier to
+    create and parse events.
+    """
+
+    def __init__(self, data=None, mimetype=None):
+        """
+        Create a new Event from a mimetype and data.
+
+        Parameters
+        ----------
+        data : bytes
+            The data to use for the event.
+        mimetype: str or int
+            The mimetype of the data (e.g. "APPLICATION_JSON").
+        """
+
+        if not data:
+            raise ValueError("no data provided")
+
+        if not mimetype:
+            raise ValueError("no mimetype provided")
+
+        # TODO: Support traditionally formatted mimetypes (e.g. application/json)
```

Comment:
This might be another good one for the GH issues in case there are external contribs looking for things to work on?

Reply:
I'm guessing we can mostly port over the code in the Go SDK?